### PR TITLE
use iOS module, use getPrice button

### PR DESCRIPTION
**Pull request by diarmuidmcg**

### CODE REVIEW

**Comment by diarmuidmcg** [comment]
* **#21** <a href="https://app.graphite.dev/github/pr/miamtech/MarmitonDemoIOSMealz/21?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @diarmuidmcg and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: MarmitonDemoIOSMealz/Features/MyBasketViewController.swift [hunk 1]
```diff
@@ -5,7 +5,7 @@
 //  Created by Diarmuid McGonagle on 04/04/2024.
 //
 
-import MealzNaviOSSDK
+import MealziOSSDK
```
Comment (walerowicz-miam) [inline MarmitonDemoIOSMealz/Features/MyBasketViewController.swift:8]: ça avait été fait et mergé via la branche [feature/merge_sdks](https://github.com/miamtech/MarmitonDemoIOSMealz/tree/feature/merge_sdks)

File: MarmitonDemoIOSMealz/Features/StandaloneRecipeDetailsViewController.swift [hunk 2]
```diff
@@ -6,7 +6,7 @@
 //
 
 import UIKit
-import MealzNaviOSSDK
+import MealziOSSDK
```
Comment (walerowicz-miam) [inline MarmitonDemoIOSMealz/Features/StandaloneRecipeDetailsViewController.swift:9]: idem : [feature/merge_sdks](https://github.com/miamtech/MarmitonDemoIOSMealz/tree/feature/merge_sdks)

File: MarmitonDemoIOSMealz/MealzViewConfig.swift [hunk 3]
```diff
@@ -9,8 +9,6 @@ import Foundation
 import MarmitonUIMealzIOS
 import mealzcore
 import MealziOSSDK
-import MealzNaviOSSDK
-import MealzUIiOSSDK
```
Comment (walerowicz-miam) [inline MarmitonDemoIOSMealz/MealzViewConfig.swift:13]: idem : [feature/merge_sdks](https://github.com/miamtech/MarmitonDemoIOSMealz/tree/feature/merge_sdks)

File: MarmitonDemoIOSMealz/Features/MyBasketViewController.swift (hunk 1 above)
Comment (diarmuidmcg) [reply MarmitonDemoIOSMealz/Features/MyBasketViewController.swift]: oui j'ai rebased, merci

**Review by antonin-francois — APPROVED** [review]

**Pull request merged**